Git homework

OleksUMD opened the pull request:
Homework is ready to be checked and merged. All tests are passed.

OleksUMD committed:
Homework #1. All tests are passed successfully

OleksUMD committed:
"print(Done)" was added to tests.py

OleksUMD committed:
"is_word_in_text" function

Split method was added in order to separate whole words.

dimigor commented on git/homework.py:
```diff
@@ -15,23 +15,29 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
     In another case should return False
     """
-    pass
+    if first == second:
```
you do not need this check, operators like `==, is, in` already returns Bool value

OleksUMD replied:
According to the task the function must check values of two objects. As far as I know "is" compares objects via theirs memory address. For instance,
d = {'a': 3, 'b':4}, d1= {'a': 3, 'b':4}
d == d1 -> True
but
d is d1 -> False # because these are different objects.
I will change the code according to your suggestion.

dimigor commented on git/homework.py:
```diff
@@ -97,14 +109,14 @@ def is_word_in_text(word: str, text: str) -> bool:
 
     """
-    pass
+    return word in text.split()
```
you do not need `.split()`. `split` converts string to list, operator `in` searches substring in str value too

OleksUMD replied:
Done!

OleksUMD committed:
Update homework

Operator '==' was replaced by 'is' in 'is_two_object_has_same_value' function.
The function 'is_word_in_text' checks for word without splitting a text to a list.

AleksMD commented:
Code was improved according to collaborators' requirements.

yuriimchg commented on git/homework.py:
```diff
@@ -15,23 +15,29 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
     """
-    pass
+    if first is second:
+        return True
```
тут не потрібно `if / else`, оскільки порівняльні стейтменти повертають булеві значення

OleksUMD replied:
Логічно) Дякую!

OleksUMD replied:
Виконав усі вищевказані рекомендації. Прогнав через flake8!

yuriimchg commented on git/homework.py:
```diff
@@ -78,7 +87,10 @@ def multiple_ints_with_conversion(first_value: Any, second_value: Any) -> int:
-    pass
+    try:
+        return int(first_value) * int(second_value)
+    except (ValueError, TypeError):
```
:+1:

yuriimchg reviewed (changes requested):
привіт
домашнє виглядає класно, але є два зауваження:

1. в перших завданнях не потрібно використовувати `if / else`.
2. варто пройтися лінтером по синтаксису, оскільки пайтон має деякі стандарти для покращення читабельності коду (PEP8). Раджу для цього використати flake8, pylint або autopep8. 
Детальніше про PEP-8: https://www.python.org/dev/peps/pep-0008/

OleksUMD committed:
Code style was improved and checked by flake8. if/else statements were simplified where neccessary.

OleksUMD committed:
Code style was improved and checked by flake8. if/else statements were simplified where neccessary.

yuriimchg commented on git/homework.py:
```diff
@@ -15,23 +16,27 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
+
+    return first == second
+
+
```
тут тільки 2 лінійки потрібно

yuriimchg reviewed (dismissed):
все окей
вітаю з виконаним домашнім :+1:

OleksUMD committed:
Updated spaces in the first task

Two empty row were deleted.
